scripts/sizeSnapshot/webpack.config.js

```diff
@@ -53,67 +73,30 @@ async function getSizeLimitBundles() {
       webpack: true,
       path: 'packages/material-ui-system/build/esm/index.js',
     },
+    ...coreComponents,
     {
-      name: 'colorManipulator',
-      webpack: true,
-      path: 'packages/material-ui/build/esm/styles/colorManipulator.js',
-    },
-    {
-      // why we use esm here: https://github.com/mui-org/material-ui/pull/13391#issuecomment-459692816
-      name: 'Button',
-      webpack: true,
-      path: 'packages/material-ui/build/esm/Button/index.js',
-    },
-    {
-      // vs https://bundlephobia.com/result?p=react-modal
-      // vs https://bundlephobia.com/result?p=@reach/dialog
-      name: 'Modal',
+      name: '@material-ui/core/styles/createMuiTheme',
       webpack: true,
-      path: 'packages/material-ui/build/esm/Modal/index.js',
+      path: 'packages/material-ui/build/esm/styles/createMuiTheme.js',
     },
     {
-      // vs https://bundlephobia.com/result?p=react-popper
-      name: '@material-ui/core/Popper',
+      name: 'colorManipulator',
       webpack: true,
-      path: 'packages/material-ui/build/esm/Popper/index.js',
+      path: 'packages/material-ui/build/esm/styles/colorManipulator.js',
     },
+    ...labComponents,
     {
-      // vs https://bundlephobia.com/result?p=react-responsive
-      // vs https://bundlephobia.com/result?p=react-media
       name: '@material-ui/core/useMediaQuery',
```

Comment:
Do we need to keep this configuration?

Reply:
We only collect components automatically. If we would collect anything we would need to manually exclude test-utils and styles.

Reply:
Make sense 👍